engines/python/src/main/java/ai/djl/python/engine/PyModel.java

```diff
@@ -165,6 +165,8 @@ public void load(Path modelPath, String prefix, Map<String, ?> options) throws I
                 } else if ("nc".equals(manager.getDevice().getDeviceType())
                         && pyEnv.getTensorParallelDegree() > 0) {
                     entryPoint = "djl_python.transformers_neuronx";
+                } else if ("TRT-LLM".equals(Utils.getenv("LMI_BACKEND"))) {
```

Comment:
@sindhuvahinis

Reply:
that doesn't cover the case when customers do not want to use rolling batch. Also, I think there's an effort to make rolling_batch as boolean property